spec/models/miq_queue_spec.rb

```diff
@@ -265,9 +265,11 @@
         :args        => ['rq_message', 1, ["A", "B", "C"], 'AUTOMATION', 'gp', 'warn', 'automate message', 'ae_fsm_started', 'ae_state_started', 'ae_state_retries'],
         :zone        => @zone.name,
         :role        => 'automate',
-        :msg_timeout => 60.minutes
+        :msg_timeout => 60.minutes,
+        :no_such_key => 'Does not exist'
```

Comment:
@jrafanie 
This basically adds a column that doesn't exist in the MiqQueue table and makes sure that we don't choke on it.

The reason for it, in our spec for requeue, we never see the custom attributes show up when we call update_attributes.